docs/usage/mounted-host-folders.md

```diff
@@ -0,0 +1,10 @@
+## Mounted host folders
+Some of drivers will mount a host folder within the VM so that you can easily share files between the VM and host.  These are no configurable at the moment and different for the driver and OS you are using.  Note: Host folder sharing is not implemented in the KVM driver yet.
+
```

Comment:
These are no configurable at the moment and different for the driver and OS you are using. > These are not configurable at the moment and different for the driver and the OS you use.  
Note: > **Note:**